### PR TITLE
Pittsburgh scrapers

### PR DESCRIPTION
**Pull request by jtotoole**
These borrow a good bit from the Chicago scrapers. Thanks!

cc @fgregg

### CODE REVIEW

File: pittsburgh/__init__.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        yield org
+
+        standing_committee = Organization(name="Standing Committee", classification="committee")
+        standing_committee.add_source("http://pittsburghpa.gov/council/standing-committees", note="web")
+        yield standing_committee
+
+        mayor = Organization(name="Mayor", classification="executive")
+        mayor.add_post("Mayor", "Mayor", division_id="ocd-division/country:us/state:pa/place:pittsburgh")
+        mayor.add_source("http://pittsburghpa.gov/mayor/index.html", note="web")
+        yield mayor
```
Comment (jtotoole) [inline pittsburgh/__init__.py]: One thing I wasn't sure about with the `Mayor` org: Our last two mayors (William Peduto and Luke Ravenstahl) were council members before being elected, so the scraper throws an error asking them to be disambiguated if I register them with their mayoral terms/posts in this file. Maybe that's fine?

**Comment by fgregg** [comment]
Is this ready for review, @jtotoole?

**Comment by jtotoole** [comment]
Yup!

**Comment by jtotoole** [comment]
The import works for me locally and the data seems sensible; it just throws a couple warnings occasionally during the `events` import because bills sometimes show up on the agenda that don't yet have their own entries in Legistar.

File: pittsburgh/__init__.py (hunk 1 above)
Comment (fgregg) [reply pittsburgh/__init__.py]: Addin their birth year should do it.

File: pittsburgh/bills.py [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+                if text["MatterTextPlain"]:
+                    bill.extras["plain_text"] = text["MatterTextPlain"]
+
+                if text["MatterTextRtf"]:
+                    bill.extras["rtf_text"] = text["MatterTextRtf"].replace(u"\u0000", "")
+
+            yield bill
+
+ACTION = {"Committee Report":
+              {"ocd" : None, "order" : 0},
```
Comment (fgregg) [inline pittsburgh/bills.py]: what is "order" doing for you hear? Doesn't seem to be used in the code elsewhere?

File: pittsburgh/people.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    # Override method in LegistarAPIPersonScraper class to pull in contact
+    # info from call to /persons/PersonId endpoint
+
+    def person_sources_from_office(self, office):
+        person_api_url = (self.BASE_URL +
+                          "/persons/{OfficeRecordPersonId}".format(**office))
+        response = self.get(person_api_url)
+        person_api_response = self.get(person_api_url).json()
+
+        return person_api_url, person_web_url, person_api_response
```
Comment (fgregg) [inline pittsburgh/people.py]: person_web_url is not defined. This is preventing me from running locally.

**Comment by fgregg** [comment]
> The import works for me locally and the data seems sensible; it just throws a couple warnings occasionally during the events import because bills sometimes show up on the agenda that don't yet have their own entries in Legistar.

This possible change would help with that, @jtotoole  https://github.com/opencivicdata/python-legistar-scraper/issues/92#issuecomment-506016631

File: pittsburgh/bills.py (hunk 2 above)
Comment (jtotoole) [reply pittsburgh/bills.py]: Ah sorry about that, that's some lazy aping of the Chicago `bills` scraper:
https://github.com/opencivicdata/scrapers-us-municipal/blob/master/chicago/bills.py#L235

I'm realizing, however, that I don't have the `sort_actions` function that draws on that field: 

https://github.com/opencivicdata/scrapers-us-municipal/blob/master/chicago/bills.py#L9-L15

Do you think `sort_actions` is worth adding, or can this scraper live without it? If the former I assume I'd want to drop it here:

https://github.com/opencivicdata/scrapers-us-municipal/pull/278/files/6eb00eeea539cf8777e85f7dc480248fd6904ee8#diff-cc73d0148d2b9fd079236552b7f45cd1R50

File: pittsburgh/__init__.py (hunk 1 above)
Comment (jtotoole) [reply pittsburgh/__init__.py]: Unfortunately still getting errors. If I do the following:

```
        peduto.add_term("Mayor",
                        "executive",
                        start_date=datetime.date(2014, 1 ,6),
                        birth_date=datetime.date(1964, 10, 30),
                        appointment=True)
```
The import fails with `TypeError: add_term() got an unexpected keyword argument 'birth_date'`.

When I try `peduto = Person(name="William Peduto", birth_date=datetime.date(1964, 10, 30))`, I get `pupa.exceptions.SameNameError: multiple people with same name "William Peduto" in Jurisdiction - must provide birth_date to disambiguate`

Is there another way of adding `birth_date` I'm missing?

File: pittsburgh/bills.py (hunk 2 above)
Comment (fgregg) [reply pittsburgh/bills.py]: you can add it later.

**Comment by jtotoole** [comment]
Found a funny little bug with `legislative_sessions` that didn't present when I first made this PR—gonna fuss with that today and will update once everything looks good

**Comment by jtotoole** [comment]
Alrighty, import is looking good locally now. I decided not to bother with `sort_actions` for the moment.

**Comment by jtotoole** [comment]
Thanks Forest!

**Pull request merged**